subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -3138,6 +3154,12 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      "0:Disable, 1:Optional, 2:Required\n"
 		      "-b --band=<band> (2 -2.6GHz, 5 - 5Ghz, 6 - 6GHz)\n"
 		      "-m --bssid=<BSSID>\n"
+		      "-i --ignore-broadcast-ssid=<type>. Hide SSID in AP mode.\n"
+		      "0: disabled (default)\n"
+		      "1: send empty (length=0) SSID in beacon and ignore probe request for "
+		      "broadcast SSID.\n"
+		      "2: clear SSID (ASCII 0), but keep the original length and ignore "
+		      "probe requests for broadcast SSID.\n"
```

Comment:
any specific use case for sending the length?

Reply:
This is aligned with hostapd configurations. The hostapd command "ignore_broadcast_ssid" support such configurations and I think those are from wireless standard.

Reply:
Thanks. Found the reason in the hostapd conf comments, `(this may be required  with some clients that do not support empty SSID)`